pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -191,9 +193,18 @@ async def _create_or_update_pull_request(self, assembly_definition: OrderedDict)
         base = self.group
         if self.runtime.dry_run:
             self._logger.warning("[DRY RUN] Would have created pull-request with head '%s', base '%s' title '%s', body '%s'", head, base, title, body)
+            if self.auto_trigger_build_sync:
+                self._logger.warning("[DRY RUN] Would have triggered build sync with the PR assembly definition since TRIGGER_BUILD_SYNC was enabled")
             d = {"html_url": "https://github.example.com/foo/bar/pull/1234", "number": 1234}
             result = namedtuple('pull_request', d.keys())(*d.values())
             return result
+
+        if self.auto_trigger_build_sync:
+            self._logger.info("Triggering build sync")
+            build_version = self.group.split("-")[1]  # eg: 4.14 from openshift-4.14
+            await start_build_sync(build_version=build_version, doozer_data_gitref=f"{branch}", assembly=self.assembly,
```

Comment:
No need for the f-string, so `doozer_data_gitref=branch`

Reply:
fixed and re pushed